### PR TITLE
Updated readme with instructions for docker engine and updated path f…

### PR DESCRIPTION
**Pull request by thompson318**
…or running tests (#27)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# mirsg.infrastructure Ansible Collection
+# mirs
+
+.infrastructure Ansible Collection
```
Comment (p-j-smith) [inline README.md:3]:
```suggestion
# mirsg.infrastructure Ansible Collection
```

**Review by drmatthews — APPROVED** [review]
Thanks @thompson318!

**Review by p-j-smith — COMMENTED** [review]
Thanks! just spotted one typo

File: README.md (hunk 1 above)
Comment (thompson318) [reply README.md:3]: Whoops, not sure what happened there, thanks for spotting. This has been merged so I've created a new PR https://github.com/UCL-MIRSG/ansible-collection-infra/pull/29

**Pull request merged**